src/types/Health.ts

```diff
@@ -332,5 +332,11 @@ export type NamespaceServiceHealth = { [service: string]: ServiceHealth };
 export type NamespaceWorkloadHealth = { [workload: string]: WorkloadHealth };
 
 export type WithAppHealth<T> = T & { health: AppHealth };
+
+export const hasHealth = <T>(val: T): val is WithAppHealth<T> => {
+  return !!val['health']['requests'];
```

Comment:
would it work with just `!!val['health']` ?
I know all kind of health today does have `requests` field, but just in terms of logic it seems to me that checking for presence of "health" field would be more correct?

Also, function signature says `val is WithAppHealth<T>`, shouldn't it be `val is WithHealth<T>` instead?